vignettes/tables.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Summary Table Creation"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Summary Table Creation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(timber)
+```
+
+In this section, we write a script to create a risk-difference table from the 
+[PHUSE Test Data Factory ADaM dataset](https://github.com/phuse-org/TestDataFactory), 
+using the [`Typlr`](https://atorus-research.github.io/Tplyr/) package. We use 
+the `timber` package to generate a log upon the script execution for 
+reproducibility and traceability.
+
```

Comment:
Should we have little section about installing timber and either axecuting with addin or the function?  If they use the addin they don't have to library(timber).  Or if they just do timber::axecute then they don't have library(timber) either.  Just thinking if someon copied this sript they wouldn't be able to axecute since timber is hidden here.